packages/lesswrong/server/callbacks/votingCallbacks.ts

```diff
@@ -72,10 +72,6 @@ voteCallbacks.cancelAsync.add(async function cancelVoteCount ({newDocument, vote
   }
 });
 
-voteCallbacks.castVoteAsync.add(async function updateNeedsReview (document: VoteDocTuple) {
-  return triggerReviewIfNeeded(document.vote.userId)
-});
```

Comment:
Wait is this a big change or is it just removing a redundancy?

Reply:
We had previously removed the actual part of the trigger that dealt with voting (it used to be if a user voted 20 times they got added to the queue, but in practice this didn't help us notice any mysterious voting patterns), so this is removing a vestigial thing

Reply:
:ty: